src/Resources/Resources/help/Get-AzResourceManagementPrivateLink.md

```diff
@@ -0,0 +1,154 @@
+PS C:\> Get-AzResourceManagementPrivateLinks
+
+
+Id                         : /subscriptions/aeb49941-36c3-4e7c-9ffd-16ba89d33ec4/resourceGroups/PrivateLinkTestRG/provi
+                             ders/Microsoft.Authorization/resourceManagementPrivateLinks/NewPL
+Type                       : Microsoft.Authorization/resourceManagementPrivateLinks
+Name                       : NewPL
+Location                   : centralus
+PrivateEndpointConnections : {}
+
+Id                         : /subscriptions/aeb49941-36c3-4e7c-9ffd-16ba89d33ec4/resourceGroups/PrivateLinkTestRG/provi
+                             ders/Microsoft.Authorization/resourceManagementPrivateLinks/NewPL2
+Type                       : Microsoft.Authorization/resourceManagementPrivateLinks
+Name                       : NewPL2
+Location                   : centralus
+PrivateEndpointConnections : {}
+```
+
+Gets all of the resoure management private links at the subscription scope.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PrivateLinkName
+The name of the private link.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+The name of the resource group.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Confirm
+Prompts you for confirmation before running the cmdlet.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases: cf
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -WhatIf
+Shows what would happen if the cmdlet runs.
+The cmdlet is not run.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases: wi
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### None
+
+## OUTPUTS
+
+### Microsoft.Azure.Commands.Resources.Models.PrivateLinks.PSResourceManagementPrivateLink
+
+## NOTES
+
+## RELATED LINKS
```

Comment:
Please link related cmdlet here, here is an example: https://github.com/Azure/azure-powershell/blob/81bb462093da3ed8bf6073e0a6233dd03a15a473/src/Accounts/Accounts/help/Get-AzContext.md?plain=1#L137